java/README.md

```diff
@@ -49,17 +49,36 @@ In the `tasks.withType(JavaCompile)` section, we pass some configuration options
 The properties file has the following template: 
 
 ```
-treatedMethods=treated,flagEnabled
-controlMethods=flagDisabled
-emptyMethods=enableFlag,disableFlag
-treatmentGroupMethods=isToggleInGroup
-annotations=FlagTesting
-linkURL=<provide_your_url>
+{
+  "piranhaMethodProperties":
+    [
+      {
+        "methodName": "isToggleEnabled",
+        "flagType": "treated",
+        "returnType": "boolean",
+        "receiverType": "com.uber.piranha.XPTest"
+      },
+      {
+        "methodName": null,
```

Comment:
why the presence of `nulls` here?

Reply:
See my comment on `properties.json`: https://github.com/uber/piranha/pull/39#discussion_r430159803